code/online_challenges/src/codechef/STONES/STONES.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <string.h>
+
+int main(void) 
+{
+    int t;
+    scanf("%d", & t);
+    while (t--) 
+    {
+        char s[100], r[100];
+        int count = 0, i, j;
+        scanf("%s", s);
+        scanf("%s", r);
+        for (j = 0; j < strlen(r); ++j) 
```

Comment:
This is extremely inefficient. `strlen` computes the length of the string every iteration. Store the length in a variable.